src/shared/exec-util.h

```diff
@@ -14,24 +14,14 @@ enum {
         _STDOUT_CONSUME_MAX,
 };
 
-typedef enum {
-        EXEC_DIR_NONE                 = 0,      /* No execdir flags */
+typedef enum ExecDirFlags {
         EXEC_DIR_PARALLEL             = 1 << 0, /* Execute scripts in parallel, if possible */
         EXEC_DIR_IGNORE_ERRORS        = 1 << 1, /* Ignore non-zero exit status of scripts */
         EXEC_DIR_SET_SYSTEMD_EXEC_PID = 1 << 2, /* Set $SYSTEMD_EXEC_PID environment variable */
         EXEC_DIR_SKIP_REMAINING       = 1 << 3, /* Ignore remaining executions when one exit with 77. */
         EXEC_DIR_WARN_WORLD_WRITABLE  = 1 << 4, /* Warn if world writable files are found */
 } ExecDirFlags;
 
-typedef enum ExecCommandFlags {
```

Comment:
Not a strict rule, but we typically declare types earlier in header. So, I'd like to keep this here.